### PR TITLE
Merging in Style Inversions

### PR DESCRIPTION
**Pull request by jomogalla**

### CODE REVIEW

File: scripts/app.js [hunk 1]
```diff
@@ -720,6 +730,7 @@
 				this.payPeriodView = false;
 				this.metricsView = false;
 				this.starredCasesView = false;
+				this.settingsView = false;
```
Comment (thomasguerena) [inline scripts/app.js:732]: duped initializer, see line 745.

Comment (jomogalla) [reply scripts/app.js:732]: how is this one a dupe?

**Comment by thomasguerena** [comment]
@jomogalla The "add interval" form is no longer anchored to the bottom of the page. Looks accidental.

File: scripts/app.js (hunk 1 above)
Comment (thomasguerena) [reply scripts/app.js:732]: On line 733 `this.settingsView = false`.
Then on line 745 `this.settingsView = false;`

**Pull request merged**